### PR TITLE
adds July content to landing page and archive page for June

### PR DESCRIPTION
**Pull request by hannahkane**
Addresses issues in https://github.com/mozilla/learning-networks/issues/418

**Changes proposed in this pull request:**
- creates new archive page for June community call
- adds content about upcoming call to community call landing page

### CODE REVIEW

**Comment by hannahkane** [comment]
@Pomax or @mmmavis - when you have a moment, can you please help me debug this?

**Comment by mmmavis** [comment]
@Pomax I don't quite get why we are getting the following errors on https://travis-ci.org/mozilla/learning.mozilla.org/builds/143422264

```bash
withTeachApi

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

  1) "after each" hook

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

  2) Uncaught error outside test suite

Error loading resource https://teach-api.herokuapp.com/auth/status (6). Details: SSL handshake failed

No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

I tried restarting the build but that didn't solve the problem 😦

**Comment by Pomax** [comment]
That's certainly a fun one, @cadecairos do you know if there are any cert issues with that heroku app?

**Comment by mmmavis** [comment]
getting the same Travis errors seen in https://github.com/mozilla/learning.mozilla.org/pull/1925#issuecomment-232149693

**Comment by hannahkane** [comment]
@cadecairos @mmmavis @Pomax - pinging again, because I'm not sure what to do next :)

**Comment by KGorr** [comment]
Please note that we had to change the date of the community call to Thursday, July 28. The time is the same. Thank you!

**Comment by KGorr** [comment]
Hey team! Is there an ETA of when this page will be live? I want to give our speakers a deadline to add info on the etherpad and will base that on when the page goes live. The call is next Thursday.
 @hannahkane @Pomax @mmmavis @cadecairos

File: pages/community-call-june-29.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+ var React = require('react');
```
Comment (cadecairos) [inline pages/community-call-june-29.jsx]: could we drop the leading space on this line

**Comment by cadecairos** [comment]
Travis seems to have finally fixed it's issues. I commented on one style nit, but otherwise this LGTM and can be merged.

**Comment by hannahkane** [comment]
@gideonthomas - ok, this is ready to be pushed to production whenever you can today. Up to you to decide whether to wait for @flukeout's Clubs work to land or not. If that doesn't get pushed to production until next week, that's okay, but this should go out today.

**Comment by KGorr** [comment]
Thanks, everyone! :)

**Pull request merged**